src/common/platform/posix/unix/gtk_dialogs.cpp

```diff
@@ -336,7 +338,7 @@ class ZUIListView : public ZUIWidget
 
 	void ConnectButtonPress(int *close_style)
 	{
-		g_signal_connect(G_OBJECT(widget), "button_press_event", G_CALLBACK(DoubleClickChecker), &close_style);
+		g_signal_connect(G_OBJECT(widget), "button_press_event", G_CALLBACK(DoubleClickChecker), close_style);
```

Comment:
This is intentional.  It was passing the address instead of a pointer and causing a *separate* crash once I put in the rest of the code :)